package.json

```diff
@@ -10,75 +10,51 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/digitalbazaar/isomorphic-js-lib-template"
+    "url": "https://github.com/digitalbazaar/authorize-access-token-middleware"
   },
   "bugs": {
-    "url": "https://github.com/digitalbazaar/isomorphic-js-lib-template/issues",
+    "url": "https://github.com/digitalbazaar/authorize-access-token-middleware/issues",
     "email": "support@digitalbazaar.com"
   },
   "license": "BSD-3-Clause",
   "main": "lib/index.js",
+  "scripts": {
+    "lint": "eslint .",
+    "test": "npm run lint && npm run test-node",
+    "test-node": "cross-env NODE_ENV=test mocha -r esm --preserve-symlinks -t 10000 test/**/*.spec.js"
+  },
   "files": [
     "lib/*.js"
   ],
   "module": "lib/main.js",
   "dependencies": {
-    "esm": "^3.2.25"
+    "@digitalbazaar/minimal-jwt": "^1.0.1",
+    "@interop/oauth2-errors": "^2.0.0",
+    "assert-plus": "^1.0.0",
+    "auth-header": "^1.0.0",
+    "base64url-universal": "^1.1.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.11.0",
-    "@babel/plugin-transform-modules-commonjs": "^7.10.4",
-    "@babel/plugin-transform-runtime": "^7.11.0",
-    "@babel/preset-env": "^7.11.0",
-    "@babel/runtime": "^7.11.0",
-    "babel-loader": "^8.1.0",
     "chai": "^4.2.0",
+    "chai-http": "^4.3.0",
     "cross-env": "^7.0.2",
-    "dirty-chai": "^2.0.1",
-    "eslint": "^7.6.0",
-    "eslint-config-digitalbazaar": "^2.5.0",
+    "eslint": "^7.9.0",
+    "eslint-config-digitalbazaar": "^2.6.1",
     "eslint-plugin-jsdoc": "^30.2.0",
-    "karma": "^5.1.1",
-    "karma-babel-preprocessor": "^8.0.1",
-    "karma-chai": "^0.1.0",
-    "karma-chrome-launcher": "^3.1.0",
-    "karma-mocha": "^2.0.1",
-    "karma-mocha-reporter": "^2.2.5",
-    "karma-sourcemap-loader": "^0.3.7",
-    "karma-webpack": "^4.0.2",
-    "mocha": "^8.1.0",
-    "mocha-lcov-reporter": "^1.3.0",
-    "nyc": "^15.1.0",
-    "webpack": "^4.44.1"
-  },
-  "nyc": {
-    "exclude": [
-      "tests"
-    ],
-    "reporter": [
-      "html",
-      "text-summary"
-    ]
-  },
-  "browser": {
-    "buffer": false,
-    "crypto": false,
-    "util": false
+    "express": "^4.17.1",
+    "lru-cache": "^6.0.0",
+    "mocha": "^8.1.0"
   },
   "engines": {
     "node": ">=12"
```

Comment:
Since we're not testing with node 12, I think this should just be bumped to 14.  We *could* test with 12, but I don't see any reason for that.

Reply:
Updated 37f62ff